challenges/weekend2/weekend2:brainix.py

```diff
@@ -0,0 +1,107 @@
+'''Redis DEVcember weekend 2.
+
+When I ran this Python script, I got the following output:
+
+RedisSet cardinality: 5000
+BloomFilter count: 4950
+HyperLogLog count: 5013
+
+RedisSet hits: 5000, misses: 0, time: 390 ms
+BloomFilter hits: 5000, misses: 0, time: 1434 ms
+HyperLogLog hits: 1310, misses: 3690, time: 52 ms
```

Comment:
Question for code reviewers: I know that HyperLogLogs aren't designed for membership testing, but that's it's possible with a `PFADD` hack. Is it expected for HLL membership testing to be this inaccurate? Or is there a problem with my approach or code?